README.md

```diff
@@ -14,33 +14,33 @@ At the time of writing the current release is `1.3`
 ## Installing Dependencies
 
 This project uses Docker and Docker Compose. For Ubuntu 16.04, an installation
-script `install-docker.sh` is provided for convenience.
+script `install-docker.sh`, is provided for convenience.
 
 ```bash
 $ ./install-docker.sh
 ```
 
 For platforms other than Ubuntu, follow the installation instructions on
 the [Docker Website][install docker ce]. Note - on other platforms, sudo is not
-required for docker commands.
+required for Docker commands.
 
-You can test your docker installation by running docker's built in hello-world:
+You can test your Docker installation by running Docker's built in hello-world:
 
 ```bash
 $ sudo docker run hello-world
 ```
 
 ## Running
 
-The project can be run with `docker compose`
+The project can be run with `Docker Compose`
```

Comment:
very true, my apologies, I would say to make that a link to https://docs.docker.com/compose/ but given the customer I suspect we shouldn't